src/renderer/views/app/AppView.tsx

```diff
@@ -57,7 +61,10 @@ export const AppView: React.FC = (): JSX.Element => {
   return (
     <Styled.AppWrapper>
       <Styled.AppLayout>
+        {isDesktopView && <AppUpdateView />}
         <Header />
+        {!isDesktopView && <AppUpdateView />}
```

Comment:
```ts
<AppUpdateView />
<Header />
...
```
No check for `isDesktopView` needed. _I know we discussed the other way, but this looks better in UI._